test/integration/functional_test.go

```diff
@@ -813,6 +813,58 @@ func validateProfileCmd(ctx context.Context, t *testing.T, profile string) {
 		}
 
 	})
+
+	t.Run("profile_list_light", func(t *testing.T) {
```

Comment:
how about mixing this these two integration tests with "profile_list" and then measure the duration of normal "profile list" and then measure the duration of "profile list --light" and expect it is less than the normal profile list.
( verify in the test that it at least 50% fasteR)